### DISCUSSION TITLE
Need minor help with components/ft-shaka-video-player

### DISCUSSION THREAD

**Discussion by k0zyrev** [FreeTubeApp/FreeTube#5704] [Q&A]
I'm experimenting with shaka player and I'm trying to do something when user clicks on the progress bar and jumps to another part of a video. All the help I need is for somebody to point out the line of code that handles it in the src/renderer/components/ft-shaka-video-player/ft-shaka-video-player.js so I could add console.log('click') and it would output 'click' every time I click on the progress bar.

I've really read the file through and through, I tried adding console.log in random promising looking places. I really don't get how it works. In general, I don't know neither node, nor javascript, but the thing I'm trying to do is tiny enough that even my little understanding should've been enough. Except this one thing.

Please help me and point out the right line of code? Please?

**Comment by absidue** [comment]
The progress bar is created and managed by the shaka-player dependency, so you'll want to look at their code for it: https://github.com/shaka-project/shaka-player

The file in FreeTube is a wrapper around that dependency, to load the data into it, to add FreeTube specific elements (e.g. full window and theater mode buttons) and to add FreeTube specific functionality like SponsorBlock integration, keyboard shortcuts and handling the format switching.

**Reply by k0zyrev** [reply]
Well, that means I put a stop to my attempt to implement overrides for sponsorblock skips. I was able to make it work with seeking with keyboard, so that will make do. So far it works as expected, the only issue being lack of mouse seeking support. 

You can find diff here if it's of any use https://pastebin.com/raw/cA0bxPUf
